openshift/oadp-operator
# fix: Add old datamover fields back to DPA

## Description

Revert removal of fields from DPA from https://github.com/openshift/oadp-operator/pull/1232, following the project deprecation process (only remove fields when API version updates). Still not allow users to use those fields.

## How to test

Create a DPA with vsm plugin or old datamover feature field. DPA should not reconcile and tell user to use Velero built-in datamover.
